wordpress-mobile/WordPress-Android
# Bump :gravatar to 2.0.1

This PR bumps the Gravatar SDK to the latest 2.0.0 version. 

On the outside, we made some updates to our public API but under the hood, the avatar upload endpoint was also updated to a v3 version.

Note: I have an issue with one screen during sign-up flow `SignupEpilogueFragment`. When I run `wasabi` flavor this screen automatically finishes and moves to the next one. Testing the `jalapeno` build seems to work fine. I've also asked @hamorillo to test this and it worked okay for him on both variants. Any idea what's happening? I can see the the `wasabi` build is getting some `AccountAction.PUSH_SETTINGS` updates. 

## To Test:

Avatar change flow from the `MeFragment` and `SignupEpilogueFragment`.

1. In the sign-up flow on the `SignupEpilogueFragment` screen.
2. Tap on the add avatar button, pick one and upload. 
3. Once logged in, go to the Me tab.
4. Tap on the avatar.
5. Follow the flow to update the avatar.
6. Go to Gravatar.com and make sure that the avatar was indeed set.

-----

## Regression Notes

1. Potential unintended areas of impact

    - no

-----

## PR Submission Checklist:

~- [ ] I have completed the Regression Notes.~

-----

## Testing Checklist (strike-out the not-applying and unnecessary ones):

~- [ ] WordPress.com sites and self-hosted Jetpack sites.~
~- [ ] Portrait and landscape orientations.~
~- [ ] Light and dark modes.~
~- [ ] Fonts: Larger, smaller and bold text.~
~- [ ] High contrast.~
~- [ ] Talkback.~
~- [ ] Languages with large words or with letters/accents not frequently used in English.~
~- [ ] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)~
~- [ ] Large and small screen sizes. (Tablet and smaller phones)~
~- [ ] Multi-tasking: Split screen and Pop-up view. (Android 10 or higher)~

The update shouldn't affect the UI so the above scenarios are not valid.
